fhir-parent/java.header

```diff
@@ -0,0 +1,6 @@
+^\/*
+^ \* \(C\) Copyright IBM Corp\. \d\d\d\d[,\d\d\d\d]*
```

Comment:
I'd rather keep this part loose.  We require DCO signoff for contributions but not copyright assignment.
Maybe just go with `.+` instead of expecting `IBM Corp.`?

The character group at the end looks a bit odd to me as well, should it be `( )` instead of `[ ]`?
```suggestion
^ \* \(C\) Copyright .+ \d\d\d\d(, \d\d\d\d)*
```